core/templates/dev/head/pages/exploration_player/StatsReportingService.js

```diff
@@ -103,9 +103,12 @@ oppia.factory('StatsReportingService', [
     if (!_editorPreviewMode) {
       $interval(function() {
         postStatsToBackend();
-      }, 10000);
+      }, 300000);
     }
 
+    // This method is called right after the exploration is started, whenever a
```

Comment:
Don't think the first clause applies any more.

Reply:
Done.